server/src/main/java/org/cloudfoundry/identity/uaa/oauth/jwt/SignatureVerifier.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * ****************************************************************************
+ *     Cloud Foundry
+ *     Copyright (c) [2009-2016] Pivotal Software, Inc. All Rights Reserved.
+ *
+ *     This product is licensed to you under the Apache License, Version 2.0 (the "License").
+ *     You may not use this product except in compliance with the License.
+ *
+ *     This product includes a number of subcomponents with
+ *     separate copyright notices and license terms. Your use of these
+ *     subcomponents is subject to the terms and conditions of the
+ *     subcomponent's license, as noted in the LICENSE file.
+ * ****************************************************************************
+ */
```

Comment:
This reminds me a question I had before. Is there specific copyright statement that we are supposed to use for UAA code currently?

Reply:
thanks, not wanted